Readme.md

```diff
@@ -26,18 +26,44 @@ In advance of the first app run:
   are working with your own Salesforce sandbox you would want to change most of the `SALESFORCE_*`
   variables.
 
+### Set up a connection to the Stripe test environment
```

Comment:
First time doing this if you have an existing docker setup you will need to rebuild it to bring in the new stripe CLI and its settings. After editing config files run:

```shell
$ docker-compose down
$ docker-compose up --build -d
```

And for testing with this but also generally its helpful to watch the logs - the stripe CLI will print logs as will matchbot. To watch both:

```
$ docker logs matchbot-app-1 --tail 100 --follow &
$ docker logs matchbot-stripe-cli-1 --tail 100 --follow &
```